src/services/Earthquakes.js

```diff
@@ -33,6 +33,16 @@ const getByDate = async (date) => {
   }
 };
 
+const getRealTime = async (current) => {
+  try {
+    const earthquakes = await getAll();
+
+    return earthquakes.filter(({ time }) => ((current - new Date(time).getTime()) / 1000) <= 600);
+  } catch (e) {
```

Comment:
You might as well remove the whole try/catch, since this just throws what gets caught. Let it bubble up.